client/layout/masterbar/style.scss

```diff
@@ -633,93 +566,6 @@ $masterbar-color-secondary: #101517;
 	}
 }
 
-.masterbar__item-notifications {
-	margin-right: 12px;
-
-	@include breakpoint-deprecated( '<480px' ) {
-		margin-right: 0;
-		margin-top: 0;
-	}
-
-	.gridicon {
-		fill: var( --color-text-inverted );
-		margin-top: 1px;
-
-		@media only screen and ( max-width: 782px ) {
-			flex-shrink: 0;
-		}
-	}
-
-	.gridicon + .masterbar__item-content {
-		padding: 0;
-
-		@include breakpoint-deprecated( '<480px' ) {
-			display: block;
-		}
-	}
-
-	.masterbar__item-notifications-label {
-		display: none;
-	}
-
-	.masterbar__notifications-bubble {
-		border: solid 2px var( --color-masterbar-background );
-		// stylelint-disable-next-line declaration-property-unit-allowed-list
-		border-radius: 50%;
-		display: none;
-		// stylelint-disable-next-line declaration-property-unit-allowed-list
-		font-size: 8px;
-		height: 8px;
-		letter-spacing: 0;
-		line-height: 12px;
-		margin: 0 0 0 -12px;
-		padding: 0;
-		position: absolute;
-		top: 3px;
-		left: 50%;
-		width: 8px;
-		z-index: z-index( '.masterbar', '.masterbar__notifications-bubble' );
-		transform: translateX( 1px );
-
-		// Animation
-		animation: bubble-unread-indication 0.5s linear both;
```

Comment:
Removing this animation is a regression. We'll need to keep it. You just have to adjust the `transform-origin`. Probably to `20px 4px` (it's relative to the SVG's viewbox), please ping me if you need anything SVG, I love SVGs.